src/components/data_search/DatasetFilterConstants.tsx

```diff
@@ -21,8 +21,9 @@ export const defaultFilters = (datasets: DatasetTerm[]): FiltersTypes => {
   };
 };
 
-export const generateDefaultParticipantCountValues = (datasets: DatasetTerm[]) => datasets.reduce((acc, dataset) => {
+export const generateDefaultParticipantCountValues = (datasets: DatasetTerm[]) => datasets.reduce(() => {
```

Comment:
Do you still need this outer reducer?  Does this simplify and meet the need?
```suggestion
export const generateDefaultParticipantCountValues = (datasets: DatasetTerm[]) => {
  return {
    max: datasets.reduce((max, dataset) => Math.max(max, dataset.participantCount ? dataset.participantCount : 0), 0),
    min: 0
  };
}
```

Reply:
Good point, will try that